### ISSUE TITLE
Enhanced FeatureImportances for Multidimensional Coefficients

### ISSUE THREAD

**Issue by rebeccabilbro** [DistrictDataLabs/yellowbrick#492]
**Describe the solution you'd like**
Create an enhanced version of the Feature Importances visualizer to allow the user to see the coefficients on a per-class basis.

**Is your feature request related to a problem? Please describe.**
Some estimators returns a multidimensional array for `coef_` , like scikit-learn's [`LogisticRegression`](http://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LogisticRegression.html), in anticipation that there will be multiple classes and decision function coefficients for each class. In the case of a binary classification, the shape of `coef_` is (1, n_features), for multiclass it will be (n_classes, n_features). In #490, we addressed the bug raised in #485 by simply averaging the per-class feature importances for each feature in the case of multidimensional coefficients (as with LogisticRegression). However, we could create an enhanced version of this visualizer that would allow the user to see the coefficients on a per-class basis, depicted as a stacked histogram.

**Examples**
The `fit` method now checks to see if the `feature_importances_` are a multidimensional array. We could add a param such as `hist=True` that instead of averaging would stack the importances per class using something like [this](https://stackoverflow.com/questions/16653815/horizontal-stacked-bar-chart-in-matplotlib/16654564#16654564) in our `draw` method. If we go this route, we may want to consider adding a target labels attribute to the class à la the [`ClassificationScoreVisualizer`](https://github.com/DistrictDataLabs/yellowbrick/blob/develop/yellowbrick/classifier/base.py#L67).

**Comment by zjpoh** [comment]
@rebeccabilbro If nobody is working on this, I would like to give it a try~
